hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/contract/hapi/ContractCallSuite.java

```diff
@@ -2275,6 +2296,8 @@ private HapiSpec transferZeroHbars() {
                         getAccountBalance(RECEIVER).hasTinyBars(10_000L));
     }
 
+    // TODO: add childRecordsCheck when child records creation is implemented
```

Comment:
The generic functionality to create child records exists. Is the missing piece specialized functionality of the smart contract service?

We should not remove validations because we are also using the tests to check the old code. If the tests do not pass the way they are, we cannot add the `@HapiTest` annotation for now.